pkgs/development/tools/ktfmt/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, fetchFromGitHub, jre, makeWrapper, maven }:
+
+maven.buildMavenPackage rec {
+  pname = "ktfmt";
+  version = "0.46";
+
+  src = fetchFromGitHub {
+    owner = "facebook";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-OIbJ+J5LX6SPv5tuAiY66v/edeM7nFPHj90GXV6zaxw=";
+  };
+
+  mvnHash = "sha256-pzMjkkdkbVqVxZPW2I0YWPl5/l6+SyNkhd6gkm9Uoyc=";
+
+  nativeBuildInputs = [ makeWrapper ];
+  installPhase = ''
+    mkdir -p $out/bin $out/share/ktfmt
```

Comment:
```suggestion
    mkdir -p $out/bin
```

Reply:
afaik `install` requires directories to exist and will return error if they are missing. Or are you suggesting that I should reuse the `bin` directory?

Reply:
-D already creates the parent dirs

Reply:
done